enrysrv/README.md

```diff
@@ -0,0 +1,88 @@
+# Enry Server
+
+Enry Server run as a gRPC service which expose `GetLanguage` to detect the programming language a file is written in.
+
+## How to install
+
+- To generate the vendor folder use `glide install` or `glide up` if you want to update the dependencies.
+- To regenerate `.proto` files, execute `go generate`
+- To build a binary `make build`. It generates `./bin/enrysrv`
+
+## Usage
+
+`enrysrv` can be launched as server or client. Client can choose if send the file content to the server.
+
+```bash
+$ ./bin/enrysrv server --max-message-size 200
+INFO[0000] version: build:07-27-2017_11_03_07 commit:d7f4809 
+INFO[0000] setting maximum size for sending and receiving messages to 209715200 
```

Comment:
Do you mean keep incoming requests logged at INFO level and the rest of the logged information at DEBUG level? So we can launch the service at INFO level by default